### PR TITLE
List restaurant data

### PR DESCRIPTION
**Pull request by leandrocollares**
This PR allows users to see restaurant names, rates, food types, delivery times and delivery taxes. 

Most important commits:

- Set up app with Create React App

- Assign IDs to restaurants

- Add RestaurantListScreen component

- Create RestaurantCard component to display restaurant data

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,33 +1,17 @@
...
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.restaurantList {
```
Comment (StanleySathler) [inline src/App.css:13]: There are a few methodologies that guide CSS development and classes naming convention. One of them is the [BEM - Block, Element, Modifier](http://getbem.com/introduction/). Further, you can read the [naming](http://getbem.com/naming/) page.

The best part about using both React and BEM is that, in React context, understanding what is a component and what is an element is a lot easier, as you usually do that when writing your own components.

As a plus for our project, I'd suggest you to adopt this methodology. It's not a hard methodology - you can learn it with a quick reading :pray: 

In summary, only your components' root element will be treated as a _Block_. All the other elements inside your component will be treated as an _Element_.

File: src/components/RestaurantCard.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const RestaurantCard = ({ restaurant }) => (
+  <div>
```
Comment (StanleySathler) [inline src/components/RestaurantCard.js:4]: To improve our semantic, I'd use [`article`](https://www.w3schools.com/tags/tag_article.asp) as the root element rather than `div`. I still have some questions about each tag introduced by HTML5, but I truly believe that `article` is the perfect element for this case. What do you think?

**Review by StanleySathler — COMMENTED** [review]
It's perfect, buddy! :star2: Well done! I've left a couple of comments to improve it a bit more.
